examples/inverse/plot_mne_cov_power.py

```diff
@@ -0,0 +1,121 @@
+"""
+===================================================================
+Compute source power estimate by projecting the covariance with MNE
+===================================================================
+
+We can use the MNE operator like a beamformer to get a power
+estimate by source localizing the bandpass filtered
+covariance matrix.
+
+"""
+# Author: Denis A. Engemann <denis-alexander.engemann@inria.fr>
+#         Luke Bloy <luke.bloy@gmail.com>
+#
+# License: BSD (3-clause)
+
+import os.path as op
+import numpy as np
+
+import mne
+from mne.datasets import sample
+from mne.minimum_norm import make_inverse_operator, apply_inverse_cov
+
+data_path = sample.data_path()
+subjects_dir = data_path + '/subjects'
+
+raw_fname = data_path + '/MEG/sample/sample_audvis_filt-0-40_raw.fif'
+raw = mne.io.read_raw_fif(raw_fname)  # already has an average reference
+raw.load_data()
+
+raw_empty_room_fname = op.join(
+    data_path, 'MEG', 'sample', 'ernoise_raw.fif')
+raw_empty_room = mne.io.read_raw_fif(raw_empty_room_fname)
+raw_empty_room.crop(0, 60)
+raw_empty_room.info['bads'] = ['MEG 2443']
+raw_empty_room.info['projs'] = raw.info['projs']
+
+events = mne.find_events(raw, stim_channel='STI 014')
+
+# event trigger and conditions
+event_id = dict(aud_l=1, aud_r=2, vis_l=3, vis_r=4)
+tmin = -0.2  # start of each epoch (200ms before the trigger)
+tmax = 0.5  # end of each epoch (500ms after the trigger)
```

Comment:
I would put `tmin=-1.0, tmax=1.5` like in the other example. That way, you have an equal number of data point for both the baseline and the ERS parts, which I would recommend.